### ISSUE TITLE
Update the docs

### ISSUE THREAD

**Issue by YagoGG** [PyGithub/PyGithub#551]
I just noticed that the [docs](http://pygithub.readthedocs.io/en/latest/introduction.html) linked in the readme are very outdated (e.g. the latest version in the changelog is from October 2014).

Would it be possible to update them with the latest content that is currently available in master?

**Comment by grahamlyons** [comment]
I notice this is still out of date:
```
$ curl -Is http://pygithub.readthedocs.io/en/latest/reference.html | grep Last-Modified
Last-Modified: Tue, 21 Apr 2015 15:31:01 GMT
```

What's the process for updating the docs? Could we automate it with each new release?

**Comment by sfdye** [comment]
+1. 

@jasonwhite @jayfk do you know the process to update the docs?

**Comment by jayfk** [comment]
No idea. I’m assuming there’s still a hook in place pointing to the old repo.

We probably need to create new docs on readthedocs and update the link accordingly. We could also try to get ownership of the old readthedocs project by contacting support.

**Comment by sfdye** [comment]
I am requesting ownership of the current dtd, once I have access, will set up the hook to automate this.

**Comment by sfdye** [comment]
Fixed in 1ca7d523fb

But right now it's still manual because I don't have access to the webhook setting.
